releasenotes/notes/2309-gateway-api.yaml

```diff
@@ -9,4 +9,5 @@ releaseNotes:
 - |
   **Fixed** Istio's Gateway API implementation to adhere to the Gateway API
   requirement that a `group: ""` field must be set for a `parentRef` of `kind: Service`.
-  Istio previously tolerated the missing group for Service-kind parent references.
+  Istio previously tolerated the missing group for Service-kind parent references. This
+  is a breaking change; see the upgrade notes for details.
```

Comment:
Is there an upgrade note for this? I don't think there needs to be because this is an alpha experimental feature, so users should not be using it in production.

Reply:
We plan to add an upgrade note; I think @ericvn was going to coordinate it with the RMs IIRC. Even if it's an alpha feature, we should still document breaking changes prominently IMO

Reply:
Fair enough, a heads up would be good, especially for something that both Istio and upstream gateway-api were previously doing in samples/tests but now won't work. I guess the advice is to add `group: ""` to any service-type parentRefs before moving to 1.19.